terraform/environments/dev.tfvars

```
@@ -1,2 +1,13 @@
-resource_group_name  = "rg-beurokratt-dev"
-storage_account_name = "storage-account-beurokratt-dev"
+resource_group_name     = "rg-beurokratt-dev"
```

Could this be byk, to be in line with everything else?

```suggestion
resource_group_name     = "rg-byk-dev"
```